docs/framework/magnitudes.rst

```diff
@@ -0,0 +1,119 @@
+.. namespace:: units
+
+Magnitudes
+==========
+
+The ratio of two Units of the same Dimension---say, `inches` and `centi<meters>`---is some constant
+number, which is known at compile time.  It's a positive real number---a _Magnitude_.
+
+We also use Magnitudes for _Dimensionless_ Units.  `percent` has a Magnitude of $1/100$, and `dozen`
+would have a Magnitude of $12$.
+
+Interestingly, it turns out that the usual numeric types are not up to this task.  We need
+a Magnitude representation that can do everything Units can do.  This means, among other things:
+
+1. We need _exact_ symbolic computation of the core operations of Quantity Calculus (i.e., products
+   and rational powers).
+
+2. We must support _irrational_ Magnitudes, because they frequently occur in practice (e.g.,
+   consider the ratio between `degrees` and `radians`).
+
+3. We should _avoid overflow_ wherever possible (note that `std::intmax_t` can't even handle certain
+   simple SI prefixes, such as `yotta`, representing $10^{24}$).
```

Comment:
LAtex math in RST: https://stackoverflow.com/questions/3610551/math-in-restructuredtext-with-latex.